WordPress/src/main/java/org/wordpress/android/WordPressDB.java

```diff
@@ -288,7 +279,6 @@ public static void deleteDatabase(Context ctx) {
     public void dangerouslyDeleteAllContent() {
         db.delete(MEDIA_TABLE, null, null);
         db.delete(CATEGORIES_TABLE, null, null);
-        db.delete(CommentTable.COMMENTS_TABLE, null, null);
```

Comment:
It looks like this `dangerouslyDeleteAllContent` method is called when the user logs out. Wouldn't we need to delete the existing comments anymore?

Assuming that we'll get rid of this class completely, we can maybe just do the cleanup in `WordPress.onAccountChanged`?